x/concentrated-liquidity/swaps_test.go

```diff
@@ -220,9 +246,17 @@ func (s *KeeperTestSuite) TestCalcAndSwapOutAmtGivenIn() {
 			// expectedTokenOut: 5000000000.000 + 4103425685.8205646999916265193598043375713541686 = 9103425685.8205646999916265193598043375713541686 round down = 9103.425685 usdc
 			expectedTokenIn:  sdk.NewCoin("eth", sdk.NewInt(2000000)),
 			expectedTokenOut: sdk.NewCoin("usdc", sdk.NewInt(9103425685)),
-			expectedTick:     sdk.NewInt(300952),
-			newLowerPrice:    sdk.NewDec(4000),
-			newUpperPrice:    sdk.NewDec(4545),
+			// crosses one tick with fee growth outside
+			expectedTick: sdk.NewInt(300952),
+			// crossing tick happens single time for each upper tick and lower tick.
+			// Thus the tick's fee growth is DefaultFeeAccumCoins * 3 - DefaultFeeAccumCoins
+			expectedLowerTickFeeGrowth: DefaultFeeAccumCoins.MulDec(sdk.NewDec(2)),
+			expectedUpperTickFeeGrowth: DefaultFeeAccumCoins.MulDec(sdk.NewDec(2)),
```

Comment:
Sorry, having trouble reconciling this with the comment.

Reply:
Ah thanks for the comment! Updated the comment to make this more understandable!